include/wx/charts/wxchartgrid.h

```diff
@@ -128,6 +128,9 @@ class wxChartGrid : public wxChartElement
     void Shift(double dx,double dy);
     void ChangeCorners(wxDouble minX,wxDouble maxX,
         wxDouble minY,wxDouble maxY);
+    void ChangeCorners(wxDouble min,wxDouble max,bool is_X = true);
```

Comment:
If we change the existing ChangeCorners function as explained above then we don't need this one.